### ISSUE TITLE
BI doesn't function properly

### ISSUE THREAD

**Issue by SpitFire-666** [Ademking/BetterViewer#32]
**Describe the bug**
Sorry to spam you! Really looking forward to test driving the addon! I'm getting some odd behaviour where BetterViewer loads but seems to fall over. I can only reproduce the issue on one PC 😑

**To Reproduce**
1. Open an image to launch BetterViewer

**Expected behavior**
BetterViewer should load the image normally. Instead, the background blurry image is shown, but the actual image doesn't show and the image controls do not function:

GIF recording here:

https://imgur.com/a/N65YCd0

**Screenshots**

https://imgur.com/a/N65YCd0

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: firefox
 - Version: 102.0a (Nightly)

**Additional context**

Since it's just one PC that's having the issue, definitely could be something weird on my end.

**Comment by Ademking** [comment]
Thank you @SpitFire-666 for your feedback 👍 
Can you disable all addons except "BetterViewer" and test again ? I think this issue is caused by other addons.

**Comment by irvinm** [comment]
I'm seeing something similar but in my case, as soon as I hover over any of the BetterViewer buttons, the image shows up.  (Before that, I can see the image briefly, then the blurred background being shown, and then nothing until I hover over the buttons)

I did try it in a clean profile and everything worked fine.  I have started disabling possible addon conflicts as well, but still narrowing it down on my side.

https://user-images.githubusercontent.com/979729/168647735-b4040cc2-021e-4fe5-96cc-0c5b16325051.mp4

Well drat ... I took another guess and disabled [Download Manager (S3)](https://addons.mozilla.org/en-US/firefox/addon/s3download-statusbar/) and that resolved it for me.  (It injects code to create a replica of the old status bar on the bottom of a page to show downloads which is very handy)  The problem for me now is that I really need to keep that addon (not a big fan of how FF handles download progress\notification) and that the addon isn't under active development any more.

**Comment by Metacor** [comment]
Since Download Manger (S3) isn't open source, I can't really find why there is an incompatibility.

Both extensions work together if you disable 'Enable Download Bar' *(but it seems that is the main reason you use that addon)*
![image](https://user-images.githubusercontent.com/35663695/190866617-ad37133c-55fb-4d2d-aee1-2b8a9903e6d5.png)

I Initially thought it had something to do with this error chrome gives on a BetterViewer image page when having both extensions enabled at the same time.
![image](https://user-images.githubusercontent.com/35663695/190866592-9e9950bc-ce26-411a-8e17-e500f9357abc.png)
Deleting all inline-styles from the code base didn't change anything, and disabling Content Security Policies stopped the errors from happening, but it didn't fix the issue.

**Comment by Ademking** [comment]
> [@Ademking](https://github.com/Ademking) I assume you are aware, but I see that Firefox (AMO) is still shows the old version.

@irvinm Yes, I'm aware. I haven't updated to version 2 on Firefox because it doesn't require migrating to Manifest V3 yet... I'm keeping the current version for now. 
If you want to use the Manifest V3 version on Firefox, you can download it from the Releases section.

**Comment by irvinm** [comment]
> > [@Ademking](https://github.com/Ademking) I assume you are aware, but I see that Firefox (AMO) is still shows the old version.
> 
> [@irvinm](https://github.com/irvinm) Yes, I'm aware. I haven't updated to version 2 on Firefox because it doesn't require migrating to Manifest V3 yet... I'm keeping the current version for now. If you want to use the Manifest V3 version on Firefox, you can download it from the Releases section.

The only reason I brought it up was that this issue was reported against Firefox and you had replied that it was no longer relevant with the new version being released.
